presto-docs/src/main/sphinx/release/release-0.279.2.rst

```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.279.2
+===============
+
+General Changes
+_______________
+* Fix a bug in calculating the adjusted queue size in ResourceGroupStates which could cause a larger adjusted queue size.
+* Upgrade Airlift to 0.207
```

Comment:
I would actually remove this.  It's more relevant to developers, who can easily look at the git blame to see when this upgrade occurred, but not relevant to the audience for these release notes, users/operators of Presto.